packages/notebook/src/modelfactory.ts

```diff
@@ -74,7 +82,7 @@ export class NotebookModelFactory
   ): INotebookModel {
     return new NotebookModel({
       languagePreference,
-      collaborationEnabled,
+      collaborationEnabled: collaborationEnabled && this.collaborative,
```

Comment:
You mean:

```suggestion
      collaborative: collaborationEnabled && this.collaborative,
```

Right?

Reply:
No, the attribute is called `collaborationEnabled`. I did not change them. See:

https://github.com/jupyterlab/jupyterlab/blob/49c5c0984b1ead0f7cd2ebac4a13378d3c7f8858/packages/notebook/src/model.ts#L522

https://github.com/jupyterlab/jupyterlab/blob/49c5c0984b1ead0f7cd2ebac4a13378d3c7f8858/packages/docregistry/src/default.ts#L27